packages/neuron-ui/src/utils/scriptAndAddress.ts

```diff
@@ -1,15 +1,18 @@
+import { Script } from '@ckb-lumos/base'
 import { predefined } from '@ckb-lumos/config-manager'
 import { encodeToAddress, parseAddress, generateAddress } from '@ckb-lumos/helpers'
 
 const { LINA: MAINNET, AGGRON4: TESTNET } = predefined
 
-const CONFIGS = {
+export const CONFIGS = {
```

Comment:
Unnecessary `export` because `CONFIGs` is not used elsewhere.

Reply:
`export` removed.